tests/distributed/utils.py

```diff
@@ -24,4 +25,11 @@ def get_local_usable_addr():
     port = sock.getsockname()[1]
     sock.close()
 
-    return ip_addr + ' ' + str(port)
\ No newline at end of file
+    return ip_addr + ' ' + str(port)
+
+
+def reset_envs():
```

Comment:
What's this for?

Reply:
we set env in some tests and such kind of env is global and will affect following tests such as `os.environ['DGL_NUM_SAMPLER']`. we boot the client in the main process and such kind of env will be read. For example, if I run `test_distributed_sampling.py` then `test_mp_dataloader.py`, it  works fine.  If I change the order, it failed as more clients are booted due to `DGL_NUM_SAMPLER`.